<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonLoad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>75</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>/Users/yuehbell/dev/C8_3DViewer_v1.0-0/src/models/Glock.obj</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Load Model (.obj):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBrowse">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>45</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelPos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Position:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinPosX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinPosY">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinPosZ">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelPX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>142</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>142</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPZ">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>142</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinRotY">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>188</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelRY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinRotX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>188</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelRot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rotation:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinRotZ">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>188</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelRZ">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinScY">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>238</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelSZ">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinScX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>238</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelSc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinScZ">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>238</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoHeader">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Controls:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>355</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>- W, A, S, D - Move in XZ-plane
- Space, C - Move Up/Down
- R - Start/Stop Model Rotation
- Right Mouse Button Drag - Rotate camera

Warning:
- Alt+Tab to the window if viewer doesn't appear.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>16</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioProjP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Center</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioProjC">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parallel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Projection:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editBgColor">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>#000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBgColor">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Background color:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBgColor">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>25</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLineColor">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Line color:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLineColor">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>25</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editLineColor">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>#FFFFFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPointColor">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Point color:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPointColor">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>25</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPointColor">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>#FFFFFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLineThick">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Line thickness:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinLineThick">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelLineType">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Line type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboLineType">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPointSize">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Point size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPointType">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Point type:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinPointSize">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="comboPointType">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
